db-scripts/src/main/resources/cgds.sql

```diff
@@ -793,7 +793,7 @@ CREATE TABLE `clinical_event_data` (
 CREATE TABLE `reference_genome_gene` (
     `ENTREZ_GENE_ID` int(11) NOT NULL,
     `REFERENCE_GENOME_ID` int(4) NOT NULL,
-    `CHR` varchar(4) DEFAULT NULL,
+    `CHR` varchar(5) DEFAULT NULL,
```

Comment:
I didn't find this change in `migration.sql`

Reply:
good catch! will update migration.sql.

Reply:
updated.